### ISSUE TITLE
Action menu gets stuck

### ISSUE THREAD

**Issue by seahorse41** [osmandapp/OsmAnd#13584]
### Description
The program becomes stuck, after tapping the action menu, but using back instead of the X to close it.
I mentioned this in passing in #13441 but now I made a proper video showing the bug and the workaround, (if forcing closed the program can be considered a workaround!)
[http://seahorsecorral.org/videos/tests/20220119_075157_L_Osmand_Action-stuck.mp4](url)

### How to reproduce?
Tap the action button, see the menu, tap back, exits the program back to the home screen. 
Either tap the Osm icon to load the program, or select it from the card deck, to resume where I left off. 
Now the action menu is not on screen, but the action button does not either reload the action menu, or close it from the stack/memory, as the [X] button would normally do.

### Your Environment
OsmAnd Version:4.1.11-421
Android/iOS version:7.1.2
Device model:LG H932

**Maps used (online or offline):**   
offline, but not relevant.

**Comment by Nine-Tailed** [comment]
It's good that you managed to reproduce this bug, I also met it, but did not understand what sequence of actions is needed.
https://github.com/osmandapp/OsmAnd/issues/13458

**Comment by mmh-dev** [comment]
possible dublicate https://github.com/osmandapp/OsmAnd/tree/fix_qa_mode

**Comment by seahorse41** [comment]
@mmh-dev That link is 404.
The dash should be a slash, at least.
What issue number did you mean to reference?

**Comment by dmpr0** [comment]
Hi @seahorse41!
@mmh-dev is reference similar issue in our internal issues tracker.  
The issue is fixed, you can download the nightly build to test http://download.osmand.net/latest-night-build/

**Comment by seahorse41** [comment]
@mmh-dev I tried OsmAnd-android-full.apk but that failed to install. I then tried OsmAnd-default.apk but it has a different icon, so I hesitated. I searched github for "apk" and "install" instructions to no avail, and wasn't until I looked in issues that I found #10597 suggesting to do that one anyway. I did, but it is not the same version, so have to start configuring it from scratch. I guess I'll wait until the next f-droid apk is released.
